packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.ts.snap

```diff
@@ -8,7 +8,7 @@ exports[`Collaborator ListItem component share inheritance indicators show when
         <avatar-image-stub userid="brian" user-name="Brian Murphy" width="36" class="files-collaborators-collaborator-indicator"></avatar-image-stub>
       </div>
       <div class="oc-pl-s oc-text-truncate">
-        <div class="oc-text-truncate"><span aria-hidden="true" class="files-collaborators-collaborator-name">Brian Murphy</span> <span class="oc-invisible-sr">Share receiver name: Brian Murphy (brian@owncloud.com)</span></div>
+        <div class="oc-text-truncate"><span aria-hidden="true" class="files-collaborators-collaborator-name">Brian Murphy</span> <span class="oc-invisible-sr"></span></div>
```

Comment:
Is this on purpose? The diff of the test doesn't really help me 😅

Reply:
I didn't include snapshots for all possible display-scenarios. But the Share receiver name is definitely handled by the tests (`shows the collaborator display name`).